### ISSUE TITLE
Reason Plot on Predicted Data Set

### ISSUE THREAD

**Issue by flachboard** [pycaret/pycaret#774]
Hello,

This isn't an issue. I just can't seem to find if this functionality is available. I'd like to generate a reason plot for observations in the predicted data set. It seems that I can only get this for the training data. Is this correct?

https://pycaret.org/interpret-model/

**Comment by pycaret** [comment]
@flachboard `interpret_model` only works with test/holdout set. The `observation` parameter passed in the `interpret_model` refers to the index of the test set. Hope this answers your question.

**Comment by pycaret** [comment]
@flachboard Please update your pycaret to `2.2.1`. There is a new parameter `use_train_data` available in the `plot_model` and `interpret_model` function which when set to True returns the plot on train data instead of test data as usual.

**Comment by flachboard** [comment]
Thanks so much!
